File: Applications/AspCore/NLP/ViewModels/DocumentSearchQueryViewModel.cs
using AngleSharp.Dom;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace NLP.ViewModels
{
    public class DocumentSearchQueryViewModel
    {
        public string searchQuery { get; set; }
        public List<DocumentViewModel> documents { get; set; }
    }
}

File: Applications/react/nlp/src/models/DocumentSearchQueryViewModel.ts
import { IDocument } from "./Document"

export interface IDocumentSearchQueryViewModel {
    searchQuery: string;
    documents: IDocument[];
}


export class DocumentSearchQueryViewModel {
    searchQuery: string
    documents: IDocument[]

    constructor() {
        this.searchQuery = "";
        this.documents = [];
    }
}
    
